src/DataTable.vue

```diff
@@ -255,6 +253,36 @@
 		},
 
 		computed: {
+			perPageOptions: function() {
```

Comment:
I replaced the longer functions with arrow functions. The rest looks good to me and works in my demo.